src/appimagetool.c

```diff
@@ -903,6 +911,28 @@ main (int argc, char *argv[])
                         printf("Will not guess update information since zsyncmake is missing\n");
                     }
                 }
+	    } else if(GITHUB_REPOSITORY){
+		gchar *zsyncmake_path = g_find_program_in_path ("zsyncmake");
+		if (zsyncmake_path){
+		   char buf[1024];
```

Comment:
This buffer isn't needed in the `if` clause.

Reply:
Done on https://github.com/AppImage/AppImageKit/pull/1075/commits/bf7c4c5c9a8c2fbec53756475fd2c1a3055d64f2